src/smart-app-banner/smart-app-banner.styles.ts

```diff
@@ -0,0 +1,85 @@
+import { Text, TextStyleHelper } from "../text";
+import { Button } from "../button";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import styled from "styled-components";
+import { IconButton } from "../icon-button";
+import { Icon } from "../icon";
+
+export const SmartAppBannerContainer = styled.div<{ $offset: number }>`
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    position: fixed;
+    margin: 8px 20px;
+    ${(props) => {
+        return `top: ${props.$offset}px;`;
+    }}
+    left: 0;
+    width: calc(100% - 40px);
+    min-height: 88px;
+    z-index: 9001;
```

Comment:
does the banner need to be above overlay? because that has a z-index of 99999

Reply:
banner needs to be under modal overlay